src/components/inputs/rich-text-input/README.md

```diff
@@ -0,0 +1,9 @@
+# RichTextInput
+
+## Usage
+
+```js
+import { RichTextInput } from '@commercetools-frontend/ui-kit';
+```
+
+#### Description
```

Comment:
I guess this is missing?

Reply:
Going to leave it out for the first iteration since things might change